frontend/src/metabase/lib/api.js

```diff
@@ -154,9 +162,10 @@ export class Api extends EventEmitter {
   }
 
   _makeRequest(...args) {
+    const options = args[5];
     // this is temporary to not deal with failed cypress tests
     // we should switch to using fetch in all cases (metabase#28489)
-    if (isTest) {
+    if (isTest || options.fetch) {
```

Comment:
Rather than build in support for formData in both xhr and fetch, I added a flag to allow endpoints to opt-in to fetch from the outset.